content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -32,6 +32,8 @@ which are configured in the API.
 Admission controllers may be "validating", "mutating", or both. Mutating
 controllers may modify the objects they admit; validating controllers may not.
 
+Admission controllers have limited scopes. They only act on requests to create, delete, modify or connect to (proxy), but not read.
```

Comment:
nit: I'd like the text to match the text in `reference/access-authn-authz/controlling-access.md` ?
`Admission control modules act on requests that create, modify, delete, or connect to (proxy) an object.`

Does the text above have the same meaning as:
```
Admission controllers limit requests to create, delete, modify or connect (proxy).
Admission controllers do not support read requests.
```

Reply:
Yes. I'll update as suggested.